src/lib/hooklib.c

```diff
@@ -186,12 +186,12 @@ char *abrt_run_unstrip_n(const char *dump_dir_name, unsigned timeout_sec)
      * Therefore we have a (largish) timeout, after which we kill the child.
      */
     libreport_ndelay_on(pipeout[0]);
-    int t = time(NULL); /* int is enough, no need to use time_t */
-    int endtime = t + timeout_sec;
+    time_t starttime = time(NULL);
+    time_t endtime = starttime + timeout_sec;
```

Comment:
Well to be completely pedantic, you could declare `timeout_sec` as `time_t` too in case somebody wanted to complain about the implicit conversion. But meh.

Reply:
Yeah, I was thinking about that, too, but the timeout should always be a reasonably small number and the implicit conversion should be fine.